lib/ansible/modules/system/iptables.py

```diff
@@ -474,7 +480,7 @@ def append_param(rule, param, flag, is_list):
     else:
         if param is not None:
             if param[0] == '!':
-                rule.extend(['!', flag, param[1:]])
+                rule.extend(['!', flag, param[1:].strip()])
```

Comment:
you might want a list comprehension here

Reply:
I think I might not because the type of `param` variable is string `<type 'str'>`.